helm-charts/orchestrator/templates/tekton-pipeline.yaml

```diff
@@ -115,8 +115,14 @@ spec:
     - name: build-and-push-image
       runAfter: ["flatten-workflow"]
       taskRef:
-        name: buildah
-        kind: ClusterTask
+        resolver: cluster
```

Comment:
we need to use the resolver also for the other tasks, since they are created in a different namespace and cannot be resolved by the pipeline that runs in orchestrator-gitops

Reply:
we only had 1 cluster task which was the 1.17 introduced breaking change, I'll test everything before marking the PR as ready anyway (for 1.11 which is the start of our support and 1.17 only, I will not test the one in-between)

Reply:
correct, the git-cli aren't cluster-task, but they failed to be resolved since they were created on a different namespace. perhaps this was a glitch of my installation steps. but let's verify this.